### DISCUSSION TITLE
✅ An AI tool like 10web.io to replicate a design from another website by providing a link.

### DISCUSSION THREAD

**Discussion by Soulgear99** [elementor/elementor#24318] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When seeking inspiration from another website for a design, it often consumes a significant amount of time to replicate the desired design  that I want.

### Describe the solution you'd like

It would be beneficial to incorporate this feature into Elementor, similar to how 10web allows users to replicate the design from another website using a link with an AI tool. 

https://github.com/elementor/elementor/assets/70857278/105e7267-e30b-41b2-9cf4-cd3c5633a43e

**Comment by nicholaszein** [comment]
Hello there @Soulgear99!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released in Elementor Core v3.18.0, and is called AI Web-Based Containers! 🥳 

You can learn more about the release of this feature in this [Blog post](https://elementor.com/blog/new-web-creation-experience-with-elementor-ai-for-containers/).

You can also learn how to use it in this [Help Center article](https://elementor.com/help/use-ai-to-create-containers-based-on-an-existing-page/)! 😉 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
